### DISCUSSION TITLE
How to change the color of the movable bar that appears when using the mouse to resize tiles?

### DISCUSSION THREAD

**Discussion by Sinaaaa** [i3/i3#5765] [Q&A] [answered]
How to change the color of the movable bar that appears when using the mouse to resize tiles?

**Comment by okraits** [comment]
I think it is the color for the `child_border` of the focused window:

https://i3wm.org/docs/userguide.html#client_colors

**Reply by Sinaaaa** [reply, marked as answer]
Thank you, your answer is not right, but without it I wouldn't have figured it out. 

It's just simply` client.focused > border.`  :O

**Reply by okraits** [reply]
Oh, indeed. My bad. I didn't try it out, I just looked at the userguide and thought that it must be the `child_border` color because it was the logical choice and the color was matching.
